tensorflow/contrib/bayesflow/python/ops/hmc_impl.py

```diff
@@ -477,6 +480,11 @@ def log_joint_partial(beta):
     accepted = math_ops.cast(
```

Comment:
if skip_metropolis_step:
      return new_x, acceptance_probs, -log_potential_1, -grad_1


Then you dont need additional ops.

Reply:
Should I make that with control flow(allowing for graph conditional) or leave it to be plain boolean?

Reply:
Plain boolean is ok for this. I doubt users will need to do this in a deferred way. And if they do, we can consider adding this feature at that time.